pkg/models/dashboards.go

```diff
@@ -317,6 +322,11 @@ func GetDashboardUrl(uid string, slug string) string {
 	return fmt.Sprintf("%s/d/%s/%s", setting.AppSubUrl, uid, slug)
 }
 
+// GetKioskModeDashboardUrl returns the HTML url for a dashboard in kiosk mode.
+func GetKioskModeDashboardUrl(uid string, slug string) string {
```

Comment:
can we avoid public function?  seems like this is not necessary in the models package

Reply:
I mostly follow the approach of grouping together functions/modules that have the same or a similar reason to change. This function constructs a webapp URL pointing at a dashboard, like many other functions in this file. 

Our backend changes would be limited to just a single file if we ever changed the dashboard URL structure, which seems like a good thing